src/input.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import Input from './Input';
```

Comment:
의미상으로 구분되어 지는 경우 공백을 넣어주세요.
```suggestion
import React from 'react';

import { render, fireEvent } from '@testing-library/react';

import Input from './Input';
```